src/components/Accordion/src/Accordion.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+	<div
+		:class="$s.Accordion"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div
+			:class="$s.AccordionHeader"
+			@click="toggleExpanded"
+		>
+			<div
+				:class="$s.TitleIconAligner"
+			>
+				<div :class="$s.Title">
+					<!-- @slot title of accordion -->
+					<slot name="title">
+						<m-text
+							pattern="title"
+							:size="-2"
+							text-transform="uppercase"
+						>
+							{{ title }}
+						</m-text>
+					</slot>
+				</div>
+				<div :class="$s.Icon">
+					<!-- @slot open & close icon -->
+					<slot name="icon">
+						<m-icon :name="iconName" />
+					</slot>
+				</div>
+			</div>
+			<div :class="$s.Secondary">
+				<!-- @slot secondary info, goes under title -->
+				<slot name="secondary">
+					<m-text
+						pattern="paragraph"
+						:size="-1"
+					>
+						{{ secondary }}
+					</m-text>
+				</slot>
+			</div>
+		</div>
+		<m-transition-collapse>
+			<div
+				v-if="isExpanded"
+				:class="$s.ContentWrapper"
+			>
+				<!-- @slot content to expand & collapse -->
+				<slot />
+			</div>
+		</m-transition-collapse>
+	</div>
+</template>
+
+<script>
+import { isUndefined } from 'lodash';
+import { MText } from '@square/maker/components/Text';
+import { MIcon } from '@square/maker/components/Icon';
+import { MTransitionCollapse } from '@square/maker/utils/TransitionCollapse';
+
+/**
+ * @inheritAttrs div
+ * @inheritListeners div
+ */
+export default {
+	components: {
+		MText,
+		MIcon,
+		MTransitionCollapse,
+	},
+
+	inheritAttrs: false,
+
+	model: {
+		prop: 'expanded',
+		event: 'accordion:update',
+	},
+
+	props: {
+		/**
+		 * if truthy or equal to expandKey the accordion will expand, otherwise it will collapse
+		 */
+		expanded: {
+			type: undefined,
+			default: undefined,
+		},
+		/**
+		 * used to orchestrate the collapsing & expanding of multiple accordions
+		 */
+		expandKey: {
+			type: undefined,
+			default: undefined,
+		},
+		/**
+		 * accordion title, will be overriden if title slot is used
+		 */
+		title: {
+			type: String,
+			required: false,
+			default: '',
+		},
+		/**
+		 * secondary info, will be overridden if secondary slot is used
+		 */
+		secondary: {
+			type: String,
+			required: false,
+			default: '',
+		},
+	},
+
+	data() {
+		return {
+			// empty string values are truthy for accordion
+			internalExpanded: !!this.expanded || this.expanded === '',
```

Comment:
See above re: the `|| this.expanded === ''` and prop type for `expanded` and `expandKey`.